src/components/PetCard/PetCard.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useNavigate } from 'react-router';
 import { Link } from 'react-router-dom';
+import { renderFavoriteIcon } from '../Utils/Utils';
 import './PetCard.css';
 
-export default function PetCard(props, newClass) {
-  const { pet } = props;
+export default function PetCard(props) {
+  const { pet, page, className } = props;
```

Comment:
Added a 'page' prop since the PetCard can be used for the pets page, shelter page and favorites page but with minor differences, we can use this prop to manipulate elements